clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -30,7 +30,15 @@ class HttpElasticsearchDAO private[dataaccess] (
     with StrictLogging {
 
   private[dataaccess] val httpClient = {
-    val restClient = RestClient.builder(httpHosts: _*).build()
+    val restClient = RestClient
+      .builder(httpHosts: _*)
+      // The default timeout is 100̈ms, which is too slow for query operations.
+      .setRequestConfigCallback(
+        _.setConnectionRequestTimeout(
+          ClioServerConfig.Elasticsearch.httpRequestTimeout.toMicros.toInt
```

Comment:
I think `setConnectionRequestTimeout` interprets its argument as millis, not micros: https://github.com/elastic/elasticsearch/blob/master/client/rest/src/main/java/org/elasticsearch/client/RestClientBuilder.java#L198